composer.json

```diff
@@ -8,9 +8,9 @@
     "license": "MIT",
 
     "require": {
-        "php": ">=5.5.0",
+        "php": "<=5.5.99",
```

Comment:
This would restrict the 1.x branch to PHP 5.5 only. Any reasons why not support PHP 5.3 / 5.4?